definitions/features/iptables.rb

```diff
@@ -0,0 +1,40 @@
+class Features::Iptables < ForemanMaintain::Feature
```

Comment:
I wonder if we should have separate features for `iptables`, `cron` + concern for the maintenance file, vs. having `maiintenance mode` feature, implementing all of those methods?